src/Selenium2Library/locators/elementfinder.py

```diff
@@ -4,6 +4,25 @@
 from Selenium2Library.utils import escape_xpath_value, events
 
 
+class ParserLocator(object):
```

Comment:
`LocatorParser` sounds a much better than `ParserLocator` IMHO. Also, I'd prefer helper class like this to be below the main class of this module.